requirements/requirements-python3.6.txt

```diff
@@ -176,7 +177,7 @@ lockfile==0.12.2
 Mako==1.1.2
 Markdown==2.6.11
 MarkupSafe==1.1.1
-marshmallow==3.5.1
+marshmallow==2.21.0
```

Comment:
This one is interesting, it is being downgraded?

Reply:
I think it could be because another requirement added constraint.